db/primary_migrate/20230118163820_add_non_restricted_mfa_required_prompt_skip_date_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddNonRestrictedMfaRequiredPromptSkipDateToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :non_restricted_mfa_required_prompt_skip_date, :date
```

Comment:
added this in case we wanted to do the cleaner format of adding back and removing the migration after ignored columns is merged. can delete this commit if unneeded. @zachmargolis  @aduth  @mitchellhenke

Reply:
I'd remove it, because then without this I can cherry-pick this on to the current deploy branch